### ISSUE TITLE
Skip removing old uploaded images when uploading new images

### ISSUE THREAD

**Issue by Theopaps** [carrierwaveuploader/carrierwave#2288]
I have implemented [this](https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Add-more-files-and-remove-single-file-when-using-default-multiple-file-uploads-feature)
And when I upload new images the old ones get deleted. So I'm trying the following inside the model in order to bypass this, but without any luck. It's not working as well.

`mount_uploaders :attachments, AttachmentUploader
skip_callback :commit, :after, :remove_previously_stored_avatar, :on=> :update, raise: false`

Any idea how I can upload new files and keep the old ones?

**Comment by ljuborados** [comment]
For anyone who might end up on this issue, I've done it like this: 
```ruby
class Rider < ActiveRecord::Base
  mount_uploaders :documents, RiderDocumentUploader

  def documents=(documents)
    docs_to_append = documents.map do |doc|
      uploader = RiderDocumentUploader.new(self)
      uploader.store! doc
      uploader
    end
    self[:documents] ||= []
    self[:documents] += docs_to_append.map {|uploader| uploader.file.filename}
  end
end

``` 

Not sure if this is the best way, but it works!

**Comment by mshibuya** [comment]
Closed by #2401.
